synapse_data_warehouse/synapse/dynamic_tables/V2.23.0__accessrequirement_latest.sql

```diff
@@ -0,0 +1,26 @@
+use schema {{database_name}}.synapse; --noqa: JJ01,PRS,TMP,CP01
+
+CREATE DYNAMIC TABLE IF NOT EXISTS ACCESSREQUIREMENT_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+    WITH latest_unique_rows AS (
```

Comment:
minor nit: I found this a bit confusing to read, and I noticed that they indent the `<query>` after the AS in the CREATE DYNAMIC TABLE [examples](https://docs.snowflake.com/en/sql-reference/sql/create-dynamic-table#examples). Do we want to adopt that formatting pattern?

Reply:
Yep! That works for me. Updated: https://github.com/Sage-Bionetworks/snowflake/pull/84/commits/05ff362a930047f4ea043f2b1c13013d871d8138